packages/fast-components-react-msft/webpack.config.js

```diff
@@ -62,7 +62,7 @@ module.exports = (env, args) => {
         },
         plugins: [
             new HtmlWebpackPlugin({
-                title: "FAST documentation",
+                title: "FAST Component explorer",
```

Comment:
Should we update this on the docs site as well. To get here I simply click "components". Perhaps we either update the button to say "component explorer", or we update this to just say, "FAST Components"?

Reply:
When we talk about it, it's the "FAST Component explorer" so I think this makes sense.  But, I can update the documentation site on https://microsoft.github.io/fast-dna to say "Component explorer" because on that site it's already in the context of FAST since it's a documentation site.